python-rhsm.spec

```diff
@@ -53,6 +54,18 @@ A small library for communicating with the REST interface of a Red Hat Unified
 Entitlement Platform. This interface is used for the management of system
 entitlements, certificates, and access to content.
 
+
+%package certificates
+Summary: Certificates required to communicate with a Red Hat Unified Entitlement Platform
+Group: Development/Libraries
+Version: 1.0.0
+
+%description certificates
+This package contains certificates required for communicating with the REST interface
+of a Red Hat Unified Entitlement Platform, used for the management of system entitlements
+and to recieve access to content. It does not have any dependency, and is pre-fixed as
```

Comment:
"recieve" is misspelled here.  You may want to rephrase the last sentence too: "Please note this package does not have a dependency on Python.  The name instead reflects its relationship to python-rhsm."